submissions/YuraZagor/Frogger/app.js

```diff
@@ -0,0 +1,92 @@
+const allEnemies = [];
+const allSpeeds = [200, 250, 350];
+const rangeY = [73, 156, 239];
```

Comment:
What are these magic numbers?
As a peer developer I am tasked to add another Y. How do I calculate it? Ideally I want to add a row number and let computer do what they were invented to do - calculations.
Also true for other numbers representing pixels across the entire codebase.

Reply:
Done